### PR TITLE
Chore: restructure header

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- Attempts to clean some stuff and start to restructure the header/navbar component

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
<em>Describe the steps with bullet points. Is there any external reference, link, or example?</em> 

## References

https://github.com/postcss/postcss-bem-linter/issues/82

### CODE REVIEW

File: src/components/common/Navbar/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  @include media(">=tablet") {
+    [data-store-search-input] {
+      max-width: 37.5rem;
+      order: 1;
+    }
+    .navlinks {
+      flex: 1 0 100%;
+      order: 3;
+    }
+    .cart-toggle { order: 2; }
```
Comment (eduardoformiga) [inline src/components/common/Navbar/navbar.scss:18]: Nice! Flexbox 💛 

Should we use tablet or notebook there?

**Review by eduardoformiga — APPROVED** [review]
Amazing 💛

File: src/components/common/Navbar/navbar.scss (hunk 1 above)
Comment (hellofanny) [reply src/components/common/Navbar/navbar.scss:18]: I've just checked with flip about the breakpoint, I'll wait for renatinha to update it :)

**Comment by gvc** [comment]
Closed in favor of #180

**Pull request closed without merging**